partcad/src/partcad/logging.py

```
@@ -158,9 +164,12 @@ def __exit__(self, *_args):
 
 
 class Action(object):
-    def __init__(self, op: str, package: str, item: str = None, extra: str = None):
+    def __init__(self, op: str, package: str, item: str = None, extra: str = None, transaction=None):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)